bulk-install-app/README.md

```diff
@@ -12,11 +12,12 @@
    - `BLU_DOMAIN` - the domain of the GitHub Enterprise Server (optional: only required for GitHub Enterprise Server)
    - `BLU_MAX_ORGANIZATIONS` - maximum number of successful organization installations per run of script 
      (optional: if not set, will install on all organizations)
-3. Run the script with the following command: `python3 -u main.py 2>&1 | tee -a bulk-install-app-results.txt`  
-   NOTE: Only organizations/users that the logged in GitHub user has permissions to install on will be installed.
+3. Run the script with the following command to install: `python3 -u main.py 2>&1 | tee -a bulk-install-app-results.txt`
```

Comment:
Suggestion: we should change the name of the script to be github_app_installer.py or something more descriptive? maybe in a follow up?